_posts/2024-04-17-dotnet8-template.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Introducing our new c# template for ASP.NET Core in .NET 8
+description: 
+date: 2024-04-17
+categories:
+- dotnet
+- functions
+- templates
+- postgres
+dark_background: true
+author_staff_member: han
+hide_header_image: true
+---
+
+We created a new template for C# and .NET 8.0. The template is based on the ASP.NET Core [Minimal API](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/minimal-apis?view=aspnetcore-8.0).
+
+In the past we had an official csharp template with limited features and a couple of community templates. Some of them are no longer maintained and still using an older .NET version so we decided to create a new official template.
```

Comment:
There weren't limited features.

It used the original forking mode of the OpenFaaS watchdog, which created one process per request, and was less efficient than newer templates where one process would handle many requests concurrently.

Reply:
> and a couple of community templates. Some of them are no longer maintained and still using an older .NET version so we decided to create a new official template.

There were also a number of unofficial templates adopted by the community, which were not necessarily kept up to date, or something that we could support directly.